src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,117 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    public static final double GROW_VALUE = 1.5;
+    private int maxSize = 10;
+    private int actualSize = 0;
+    private Object[] array = new Object[maxSize];
+    private Object[] tempArray;
```

Comment:
- temp array shouldn't be a field
- initialization should happen in the constructor, as mentioned in the checklist
- use type parameter instead of Object